roles/postgresql_upgrade/README.md

```diff
@@ -0,0 +1,19 @@
+theforeman.operations.postgresql_upgrade
+========================================
+
+Upgrades Postgresql for a Foreman install.
```

Comment:
```suggestion
Upgrades PostgreSQL for a Foreman install.
```

Reply:
I think this also should point out that this is only for EL8 systems with module-based PSQL, which will update 10 to 12, not any arbitrary versions.